app/javascript/components/searchBar/search-bar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import style from './search-bar.scss';
+
+const SearchBar = (props) => {
+    const searchTermPlaceHolder = "Find a restaurant";
+
+    function handleChange(event) {
+        if(props.onSearchTerm){
+            props.onSearchTerm(event.target.value);
+        }
+    }
+
+    return(
+        <div className={style.box_container}>
+            <i className="fa fa-search" aria-hidden="true"></i>
+            <input type="text" placeholder={searchTermPlaceHolder} onChange={handleChange} />
+        </div>
+    )
+}
+
```

Comment:
plenty of breaklines everywhere.
consider adding a linter to your IDE/editor.  (will be extra helpful before you actually pick up some of the team's tasks)
i would add something for both eslint and rubocop.

https://connect.we.co/display/DIGI/Linter+Setup

Reply:
👍